src/chapters/how-to-code/comments.rst

```diff
@@ -24,7 +23,7 @@ Experiment by adding and removing comments to the code.
 
 Notice that when you run the program, it still prints the phrase ``Hello,
 World``, but none of the comments appear. Also notice the blank lines left in
-the code, which are also ignored by the compiler.
+the code also ignored by the compiler.
```

Comment:
Small phrasing fixes: "...that blank lines... also get ignored...".

Reply:
What about "Also, notice the compiler also ignores the blank lines left in the code."?